utils/src/main/java/software/amazon/awssdk/utils/DateUtils.java

```diff
@@ -122,10 +121,9 @@ public static Instant parseServiceSpecificInstant(String dateString) {
         }
         try {
             BigDecimal dateValue = new BigDecimal(dateString);
-            return Instant.ofEpochMilli(dateValue.scaleByPowerOfTen(
-                    AWS_DATE_MILLI_SECOND_PRECISION).longValue());
+            return Instant.ofEpochMilli(dateValue.scaleByPowerOfTen(AWS_DATE_MILLI_SECOND_PRECISION).longValue());
         } catch (NumberFormatException nfe) {
-            throw SdkClientException.builder().message("Unable to parse date : " + dateString).cause(nfe).build();
+            throw new IllegalArgumentException("Unable to parse date : " + dateString, nfe);
```

Comment:
Should probably catch this in the unmarshallers and throw back an SdkServiceException as this is likely a result of the service returning a bad value and nothing to do with the customers input.